backend/README.md

```diff
@@ -13,13 +13,18 @@ should only need to happen once.*
 ```
 docker image rm $(docker image list -q)
 docker volume rm $(docker volume list -q)
-docker-compose build
```

Comment:
If we leave this documentation and remove --build from the commands we can save a few seconds off every up